lighthouse-core/gather/gather-runner.js

```diff
@@ -631,6 +631,11 @@ class GatherRunner {
 
       await GatherRunner.setupDriver(driver, options);
 
+      const milestone = (await driver.getBrowserVersion()).milestone;
+      if (milestone < 83) {
```

Comment:
Ideally we could warn on "older than 2 Chrome versions" or something, but we don't have a way to know what the current stable version is (unless we were to extrapolate based on the release cadence, which, AFAICT, is reliable but feels hacky).

I wonder if we could possibly extend [Browser.getVersion](https://chromedevtools.github.io/devtools-protocol/tot/Browser/#method-getVersion) to return the release date. That may be the best solution, and as a first pass we can just hardcore `83` here.

Reply:
That's a more general solution to older chrome's running against LH. I personally really like that we have specific milestone requirements scattered throughout.. Also makes it much easier for us to track the age of any particular assumption.

Reply:
I will add a TODO for now